app/models/form/question.rb

```diff
@@ -14,11 +14,11 @@ module GuidancePosition
   def initialize(id, hsh, page)
     @id = id
     @page = page
+    @guidance_position = GuidancePosition::TOP
```

Comment:
why cant this be injected like everything else? and instead must be overridden?

Reply:
It can, just setting a default here so that we don't have to set it every time (only have to set it when it's at the bottom)